v3/newrelic/trace_context_test.go

```diff
@@ -46,6 +47,47 @@ type TraceContextTestCase struct {
 	} `json:"intrinsics"`
 }
 
+func TestJsonDTHeaders(t *testing.T) {
+	type testcase struct {
+		in  string
+		out http.Header
+		err bool
+	}
+
+	for i, test := range []testcase{
+		{"", http.Header{}, false},
+		{"{}", http.Header{}, false},
+		{" invalid ", http.Header{}, true},
+		{`"foo"`, http.Header{}, true},
+		{`{"foo": "bar"}`, map[string][]string{
```

Comment:
Why is this `map[string][]string` rather than just another http.Header? I've changed them back to http.Header and all seems to work fine.

Reply:
http.Header is more correct to use, even though they're (currently) equivalent.